### PR TITLE
Port to use mdbook

### PR DESCRIPTION
**Pull request by CT075**

### CODE REVIEW

File: docs/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+smlhelp-demo.camdar.io
```
Comment (HarrisonGrodin) [inline docs/CNAME:1]: Should probably change (or potentially remove the file altogether)?

**Review by HarrisonGrodin — COMMENTED** [review]
Is it possible to get GH Pages to build automatically, so we don't have to commit the `docs/` directory?

File: docs/CNAME (hunk 1 above)
Comment (jacobneu) [reply docs/CNAME:1]: Is this file required at all?

Comment (CT075) [reply docs/CNAME:1]: It isn't, and I don't know how it got committed (probably due to my setting up the custom domain on my end)

**Pull request merged**